Makefile

```diff
@@ -46,6 +46,7 @@ check-all: check check-pypy check-noextensions
 
 typing:
 	mypy dulwich
+	mypy --strict dulwich/ssh_keygen.py
```

Comment:
There's multiple places where we invoke mypy; I'd rather enable strictness on a per-project basis, separate of this PR. If you insist, please add this in pyproject.toml so all mypy invocations use it.

Reply:
I think the problem is strict mode is either on or off per mypy invocation and can not be enabled or disabled on a per-submodule basis. I think closest one can get to "enabling strictness on a per-project basis" is to make separate invocation of mypy specifically scoped to a sub-project.

I don't think you actually want strict mode on for all mypy invocations because most invocation will also type check all other other Python files which is going to generate a bazillion errors.

Something I can cook up if you want to see it is turn on strict for all innovations and then put in a bazillion disable settings for all the submodules EXCEPT ssh_keygen (and whatever files will pass strict mode). It's kind of a pain in the neck though. But if the long-term plan is to get lots of files passing strict, it's the way to go.

Reply:
I can keep track of strictly typed files from my beautiful justfiles in the dulwich-sshsiglib union repo :wink: